src/Sulu/Bundle/MediaBundle/Resources/js/containers/MediaCollection/stores/CollectionInfoStore.js

```diff
@@ -0,0 +1,78 @@
+// @flow
+import {action, autorun, computed, observable} from 'mobx';
+import {ResourceRequester} from 'sulu-admin-bundle/services';
+import type {BreadcrumbItem, BreadcrumbItems, Collection} from '../types';
+
+const COLLECTIONS_RESOURCE_KEY = 'collections';
+
+export default class CollectionInfoStore {
+    @observable loading: boolean = false;
+    @observable collection: Collection = {
+        parentId: null,
+        breadcrumb: null,
+    };
+    @observable parentCollectionId: ?string | number;
+    disposer: () => void;
+
+    constructor(collectionId: ?string | number, locale: string) {
+        this.disposer = autorun(() => {
```

Comment:
Seems like you haven't considered https://github.com/sulu/sulu/pull/3565#discussion_r149284265

Reply:
The `autorun` would only make sense if you make the `locale` an observable, which can be set from the outside like in the `DatagridStore`. Would maybe even make sense, but then you would have to change the `createStores` method in the `MediaOverview`.